docs/integration/index.md

```diff
@@ -0,0 +1,39 @@
+# Overview
```

Comment:
Previous page had a bit more information on the context of the integration. While this pr seem to organize the integration tutorial, it also makes it less transparent on the roles of each step. As a solution, I would propose to add more information at each of the section.  Maybe adding an objection subsection to each of the sections would be nice before providing the link. For example, for the `vehicle_inteface` section it could be: "To integrate Autoware into a real vehicle, it is necessary to connect the vehicle base controller with the Autoware control module. In a nutshell, in this section we develop a ROS2 package called `vehicle_interface` that serves two purposes: 1. Receiving Autoware velocity commands through ROS2 topics, converting them into the vehicle specific commands and propagating them to the vehicle through thevehicle specific API. 2. Receiving the vehicle state through the vehicle specific API and propagating it to Autoware through ROS2 topics."